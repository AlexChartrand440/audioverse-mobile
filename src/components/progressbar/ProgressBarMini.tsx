import React from 'react'
import { View, StyleSheet } from 'react-native'
import { ProgressComponent } from 'react-native-track-player'

class ProgressBarMini extends ProgressComponent {

  render() {
    
    return (
      <View
        style={[
          {width: (this.getProgress() * 100) + '%'},
          styles.container
        ]}
      />
    )
  }

}

const styles = StyleSheet.create({
  container: {
    height: 2,
    backgroundColor: '#E53935',
  },
})

export default ProgressBarMini
