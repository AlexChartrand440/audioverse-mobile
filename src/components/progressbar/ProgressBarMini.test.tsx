import React from 'react';
import renderer from 'react-test-renderer';

import ProgressBarMini from './ProgressBarMini';

test('renders correctly', () => {
	const tree = renderer.create(<ProgressBarMini />).toJSON();
	expect(tree).toMatchSnapshot();
});
