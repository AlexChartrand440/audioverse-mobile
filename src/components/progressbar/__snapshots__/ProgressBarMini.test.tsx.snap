// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "width": "NaN%",
      },
      Object {
        "backgroundColor": "#E53935",
        "height": 2,
      },
    ]
  }
/>
`;
